traitsui/qt4/video_editor.py

```diff
@@ -102,6 +190,21 @@ class VideoEditor(Editor):
     #: The playback rate.  Negative values rewind the video.
     playback_rate = Float(1.0)
 
+    def _get_frame(self):
+        if self.control.image is not None:
+            return np_from_QImage(self.control.image)
+        else:
+            return [[[0, 0, 0]]]
+
+    def _get_image_fun(self):
+        return self.control.image_fun
+
+    def _set_image_fun(self, new_fun):
+        self.control.image_fun = new_fun
+
+    def _control_default(self):
```

Comment:
TraitsUI needs to be very careful about when Qt stuff is created - if something happened to reference the control trait before there were a QApplication then this could cause an error.  Low probability, but don't rely on defaults like this.

Reply:
Removed default constructor